.evergreen/config.yml

```diff
@@ -136,42 +156,50 @@ axes:
       - id: ubuntu-18.04
         display_name: "Ubuntu 18.04"
         run_on: ubuntu1804-test
+        variables:
+          PYTHON: "/opt/mongodbtoolchain/v3/bin/python3"
+          VENV_BIN_DIR: "bin"
 
       - id: ubuntu-20.04
         display_name: "Ubuntu 20.04"
         run_on: ubuntu2004-test
+        variables:
+          PYTHON: "/opt/mongodbtoolchain/v3/bin/python3"
+          VENV_BIN_DIR: "bin"
+      
+      - id: macos-10.14
```

Comment:
due to the small size of the macOS 10.15 fleet plus issues with the Xcode installs on those machines (see [BUILD-14623](https://jira.mongodb.org/browse/BUILD-14623)) I switched the min Swift version task to run on macOS 10.14 instead.